fdbclient/FDBTypes.h

```diff
@@ -80,7 +80,7 @@ const StorageTeamID txsTeam = UID(1, 1);
 
 // Map from storage team id to a TLog group. "tLogGroups" will be sorted so
 // that the assignment is consistent across calls/machines.
-TLogGroupID tLogGroupByStorageTeamID(std::vector<TLogGroupID>& tLogGroups, const StorageTeamID& storageTeamID);
+TLogGroupID tLogGroupByStorageTeamID(const std::vector<TLogGroupID>& tLogGroups, const StorageTeamID& storageTeamID);
```

Comment:
Several reasons: this function indicates "get tLogGroup by Storage Team ID", so from its name, it should be a pure function with no side effect; if the expected input is always sorted, we should consider storing the TLogGroupID in a set, rather than a vector.

Reply:
OK. Can you update the comment to reflect the change?

Reply:
Done

Reply:
This is a nasty bug I fixed earlier. So I think it's worth mentioning why sorting is needed.